openfe_gromacs/tests/protocols/test_gromacs_md_slow.py

```diff
@@ -0,0 +1,38 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe-gromacs
+
+import gufe
+import pytest
+from openff.units import unit as off_unit
+
+from openfe_gromacs.protocols.gromacs_md.md_methods import GromacsMDProtocol
+
+
+@pytest.mark.integration
+def test_protein_ligand_md(toluene_complex_system, tmp_path_factory):
+    settings = GromacsMDProtocol.default_settings()
+    settings.simulation_settings_em.nsteps = 10
+    settings.simulation_settings_nvt.nsteps = 10
+    settings.simulation_settings_npt.nsteps = 100
+    settings.simulation_settings_em.rcoulomb = 1.0 * off_unit.nanometer
+    settings.simulation_settings_nvt.rcoulomb = 1.0 * off_unit.nanometer
+    settings.simulation_settings_npt.rcoulomb = 1.0 * off_unit.nanometer
+    protocol = GromacsMDProtocol(
+        settings=settings,
+    )
+
+    dag = protocol.create(
+        stateA=toluene_complex_system,
+        stateB=toluene_complex_system,
+        mapping=None,
+    )
+
+    shared_temp = tmp_path_factory.mktemp("shared")
+    scratch_temp = tmp_path_factory.mktemp("scratch")
+    gufe.protocols.execute_DAG(
+        dag,
+        shared_basedir=shared_temp,
+        scratch_basedir=scratch_temp,
+        keep_shared=False,
+        n_retries=3,
+    )
```

Comment:
For a full integration test, you really want to run "this thing under normal circumstances".

What you're testing is "is this thing yielding the results I would expect".

To do that you could test:

1. Am I getting the right temperature?
2. Am I getting the right pressure?
3. Is my box volume equilibrating during the NPT phase?
4. Is my ligand sticking around the binding site as I would expect it to?
5. Is my energy reasonably well maintained?

Because you're marking it with the "this is really expensive" flag, it's perfectly ok to make this a much longer test (many more steps, etc...).